test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -311,7 +312,8 @@ private void cleanUp() {
             try {
                 writer.close();
             } catch (Throwable e) {
-                log.error("Error closing reader", e);
+                log.error("Error closing writer", e);
```

Comment:
Here and below: If you simply don't catch the exception, it will go up the stack and fail the test with the full stacktrace.

Reply:
done.